cclient/iota_client_extended_api.c

```diff
@@ -138,13 +138,18 @@ retcode_t iota_client_get_inputs(iota_client_service_t const* const serv,
   if (ret_code == RC_OK && balances_req->addresses) {
     // expect balance value is in order.
     CDL_FOREACH(balances_req->addresses, q_iter) {
-      tmp_balance = get_balances_res_balances_at(balances_res, counter);
-      if (tmp_balance >= threshold) {
+      if (out_input->total_balance < threshold) {
+        tmp_balance = get_balances_res_balances_at(balances_res, counter);
```

Comment:
What is the tmp for ? Can't you just directly add to `out_input->total_balance` ?